### DISCUSSION TITLE
2 immagini di home Assistant

### DISCUSSION THREAD

**Discussion by RobyDomo** [indomus/forum#1405] [HUB: Home Assistant]
Buon pomeriggio,
ho un RPI 4 con Home assistant core installato su docker.
Digitando il comando docker image mi compaiono:

REPOSITORY                                 TAG            IMAGE ID       CREATED         SIZE
homeassistant/raspberrypi4-homeassistant   stable         480be8cb7baf   5 weeks ago     1.1GB
ghcr.io/home-assistant/home-assistant      stable         be0394712c9f   5 weeks ago     1.07GB

Ma su docker compose il riferimento è alla 1 immagine ( homeassistant/raspberrypi4-homeassistant)

posso cancellare l'altra immagine (visto che occupa 1 GB)?
C'è un motivo perchè ce ne sono 2?
Grazie e buona giornata.

**Comment by snakuzzo** [comment]
Quando crei il container con la nuova immagine, quella vecchia non viene rimossa in automatico.
Puoi rimuovere quelle vecchie con...

`docker image prune -f`

**Reply by RobyDomo** [reply]
Grazie @snakuzzo 
Ma, a memoria, io dovrei aver creato solo 1 container con HA. 
Non ricordo di aver creato un container e poi averne creato un altro con un immagine più nuova.
Cmq potrei sbagliare, visto che la mia memoria è ballerina.
Grazie. :-)

**Reply by snakuzzo** [reply]
Di sicuro non te l'ho scaricata io...né si scaricano da sole :)

Comunque se vedi, una l'hai presa da dockerhub e l'altra da github respository. Sicuramente avrai inizialmente creato il container con quella di dockerhub per poi passare a quella di ghrc (o viceversa).

**Reply by RobyDomo** [reply]
:-) molto probabilmente è come dici tu...
Ho seguito la guida di InDomus e li è utilizzata l'immagine di GitHub. Quindi probabilmente ho installato con Github e poi per qualche motivo ho messo quella di DockerHub su Docker-compose.
A questo punto c'è differenza tra le 2 immagini oppure 1 vale l'altra?
Grazie delle spiegazioni sempre puntuali.

**Reply by snakuzzo** [reply]
sono uguali
